internal/resources/policy/constants.go

```diff
@@ -11,4 +11,17 @@ const (
 	KeyKey               = "key"
 	OperatorKey          = "operator"
 	ValuesKey            = "values"
+	ScopeKey             = "scope"
```

Comment:
If this is not being used elsewhere, can we update it to `scopeKey`?

Reply:
Used inside the security policy resource. Can't change it.

Reply:
Can you please add a link to the same, as a reference?